docs/user-guide.md

```diff
@@ -678,8 +678,9 @@ val results2 = g.pageRank.resetProbability(0.15).maxIter(10).run()
 // Run PageRank personalized for vertex "a"
 val results3 = g.pageRank.resetProbability(0.15).maxIter(10).sourceId("a").run()
 
-# Run PageRank personalized for vertex ["a", "b", "c", "d"] in parallel
-val results3 = g.pageRank.resetProbability(0.15).maxIter(10).sourceIds(Array("a", "b", "c", "d")).run()
+// Run PageRank personalized for vertex ["a", "b", "c", "d"] in parallel
+// Works only in Spark 2.1+
```

Comment:
this is true - I think we are missing some checks/handling for this in `parallelPersonalizedPageRank`